Mixpanel/Mixpanel.m

```diff
@@ -911,51 +910,39 @@ - (id)unarchiveFromFile:(NSString *)filePath
 
 - (void)unarchiveEvents
 {
-    self.eventsQueue = (NSMutableArray *)[self unarchiveFromFile:[self eventsFilePath]];
-    if (!self.eventsQueue) {
-        self.eventsQueue = [NSMutableArray array];
-    }
+    self.eventsQueue = (NSMutableArray *)[self unarchiveFromFile:[self eventsFilePath] asClass:[NSMutableArray class]] ?: [NSMutableArray array];
```

Comment:
This actually changes the logic here. We want to create an `NSMutableArray` if we were unable to load one.